src/Utils/CustomFieldShowService.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace CubeTools\CubeCustomFieldsBundle\Utils;
+
+/**
+ * This service allows to get required information from custom_fields table for showing.
+ */
+class CustomFieldShowService
+{
+    private $configReader;
+
+    public function __construct(ConfigReader $configReader)
+    {
+        $this->configReader = $configReader;
+    }
+
+    public function getDataNonempt($entity)
```

Comment:
shouldn't the method name be "getDataNonempty" ? (missing "y")

Reply:
hm, of course